src/test/helpers/helper.ts

```diff
@@ -9,3 +10,23 @@ export function mock<T>(c: new (...args: any[]) => T): T {
 export function mockObject(...propNames: string[]) {
   return td.object(propNames);
 }
+
+/**
+ * Assert if `value` contains `includes` string.
+ * @param {string} value The value that should contain the `includes` string.
+ * @param {string} includes The string that should be inside the `value`.
+ * @returns True when `value` contains `includes` string. Otherwise will throw `AssertionError`.
+ * @throws {AssertionError} Will throw `AssertionError` when assert is false.
+ * @remarks Copied from @jerone/assert-includes with permission from @jerone.
+ */
+export function assertIncludes(value: string, includes: string) {
+  const result = value.includes(includes);
+  if (result === true) return true;
```

Comment:
Is `if (result) return true;` enough as `includes` should return boolean value?

Reply:
Instead of returning `false`, I'm throwing `AssertionError`, which gets picked up by mocha.
Actually, we probably don't even have to return `true`, because it's unused.
Or we can change the code in the test to include `assert.ok`:
```
assert.ok(assertIncludes(html, expectation));
```

Reply:
> Actually, we probably don't even have to return `true`

I actually like this. Generally speaking, an assertion throws an exception if the assumption turns out to be false, but it does nothing otherwise; so "do nothing" sounds fine to me. Perhaps we can just flip the `if` condition to throw? (i.e. `if (!value.includes(...)) throw ...`

Reply:
I applied your feedback.